unblob/handlers/compression/bzip2.py

```diff
@@ -52,8 +52,16 @@ def calculate_chunk(
         self, file: io.BufferedIOBase, start_offset: int
     ) -> Optional[ValidChunk]:
 
-        self.parse_header(file, Endian.BIG)
-        end_block_offset = self.bzip2_recover(file, start_offset)
+        header = self.parse_header(file, Endian.BIG)
+        end_block_offset = -1
+        while header.compressed_magic == b"1AY&SY\x86\xc6":
```

Comment:
I would make it a CONSTANT and explain the value

https://stackoverflow.com/questions/18262703/bzip2-block-header-1aysy

Reply:
Fixed.